spring-social-core/src/main/java/org/springframework/social/connect/redis/data/SocialRedisConnectionRepository.java

```diff
@@ -0,0 +1,20 @@
+package org.springframework.social.connect.redis.data;
+
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface SocialRedisConnectionRepository extends CrudRepository<SocialRedisConnection, String> {
+
+    Iterable<SocialRedisConnection> findByProviderId(final String providerId);
+
+    Iterable<SocialRedisConnection> findByUserIdAndProviderId(final String userId, final String providerId);
+
+    SocialRedisConnection findOneByUserIdAndProviderIdAndProviderUserId(String userId, String providerId, String providerUserId);
+
+    void deleteByUserIdAndProviderIdAndProviderUserId(final String userId, final String providerId, final String providerUserId);
```

Comment:
Cool, thanks for looking at it, I will make the adjustments, add unit tests and update the dependency version from `1.7.0.RC1` to `1.7.1.RELEASE`